### DISCUSSION TITLE
How do I get the Sky Cover data from the API?

### DISCUSSION THREAD

**Discussion by sean-user** [weather-gov/api#280] [Q&A]
I would like to use the API to get the hourly Sky Cover % data. But I don't know the endpoint to use to get it. Can you send me an example of the API call to retrieve this data?

The data I'm looking for is shown at this link graphically or tabulated, but I need to get it from the API.

https://forecast.weather.gov/MapClick.php?w3u=1&w4=sky&w10u=0&w12u=1&AheadHour=0&Submit=Submit&FcstType=graphical&textField1=34.125&textField2=-118.05&site=all&unit=0&dd=&bw=

Thank you.

**Comment by jfbauer432** [comment]
There may be other ways, but this should work.

Fetch the points endpoint for your location and find the
properties.forecastGridData item and then fetch that URL.
In that you can find the sky cover at properties.skycover.values[]

For the location you indicated, these would be the URLs you would
then be accessing:

https://api.weather.gov/points/34.13,-118.04
https://api.weather.gov/gridpoints/LOX/162,46

**Comment by KTibow** [comment]
Check the NWS integration in Home Assistant.

**Comment by sean-user** [comment]
Ah, I was using https://api.weather.gov/gridpoints/LOX/162,46/forecast which didn't have this data. I now see it in the call without /forecast.

Thank you all so much!

**Comment by scadergit** [comment]
We hope to enhance the forecast endpoint before long...just tackling some more critical priorities first.
